coordinator/src/lib.rs

```diff
@@ -17,6 +17,8 @@
 extern crate codechain_logger as clogger;
 extern crate log;
 
+#[macro_use]
+mod config_common;
```

Comment:
`config_common` is a good name, but maybe `desc_common` is also a good name?

Reply:
Good. The module contains common code of app descriptor and link descriptor.